code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -231,7 +231,12 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	layer = FLY_LAYER
 	appearance_flags = TILE_BOUND
 
-/obj/effect/overlay/gas/New(state, alph)
+/obj/effect/overlay/gas/New(state, alph, gay)
 	. = ..()
 	icon_state = state
 	alpha = alph
+	if(gay)
+		icon_state = "pridesma"
```

Comment:
All gasses become plasma when it's Gay?

Reply:
it's only gay if the gas path is plasma and it's june